app/models/hotel.rb

```diff
@@ -1,3 +1,8 @@
 class Hotel < ApplicationRecord
   has_many :rooms
-end
\ No newline at end of file
+  has_many :reservations, through: :rooms
```

Comment:
Awesome work extending these associations! We'll talk more about this after break on your group project

Reply:
You could even go further and say `has_many :guests, through: :rooms` since you've set up an association between guests and rooms in the Room model. 

That way, for US 3, you could just do, from a hotel instance method: `self.guests.distinct.count`